cli/cli/build.gradle.kts

```diff
@@ -44,7 +44,8 @@ dependencies {
   // Include these FileIO implementations, as those are necessary to initialize
   // RESTCatalog *sigh*. Those FileIO's aren't functional, because the necessary
   // cloud specific clients (e.g. awssdk) are missing.
-  implementation("org.apache.iceberg:iceberg-aws")
+  implementation("org.apache.iceberg:iceberg-aws:1.6.1")
+  implementation("org.apache.iceberg:iceberg-aws-bundle:1.6.1")
```

Comment:
Please `gradle/libs.versions.toml` to manage library versions.

Reply:
And while at it, I thinkw e can go ahead and add the three bundles:

```kotlin
  // Include these FileIO implementations, as those are necessary to initialize
  // RESTCatalog *sigh*.
  implementation("org.apache.iceberg:iceberg-aws")
  implementation("org.apache.iceberg:iceberg-aws-bundle")
  implementation("org.apache.iceberg:iceberg-azure")
  implementation("org.apache.iceberg:iceberg-azure-bundle")
  implementation("org.apache.iceberg:iceberg-gcp")
  implementation("org.apache.iceberg:iceberg-gcp-bundle")
```

Reply:
What we need to assess now is the jar size... it goes from 25Mb to 120Mb 🙄

Reply:
> What we need to assess now is the jar size... it goes from 25Mb to 120Mb 🙄

That's quite too big. I doubt a fix would need the whole Iceberg bundle, rather only the necessary dependencies.